src/robinhood.py

```diff
@@ -22,7 +22,7 @@ def check_login_time(cls):
 
         time_delta = abs(datetime.strptime(cls.login_time, format) - datetime.strptime(current_time, format))
 
-        # Relogin after 22 hours
```

Comment:
We should probably undo this change, the term `log` is usually used in the context of saving information for crashes or status reports.